### ISSUE TITLE
Execuse me, where is the source code?

### ISSUE THREAD

**Issue by shootingstar123** [dinhtuantran/triple-sigmoid#1]
Fight Against Academic Fraud

**Comment by shootingstar123** [comment]
In these experiments, two neural network models imple- mented using the Keras library were utilized (the source code and experimental setting details are available at https://github.com/dinhtuantran/triple-sigmoid).
on ieee publication

**Comment by dinhtuantran** [comment]
I am sorry. We are attempting to clean up the source code to make it simpler to read before publication. This week, it will be accessible.
